devimint/src/external.rs

```diff
@@ -373,6 +374,60 @@ pub async fn open_channel(bitcoind: &Bitcoind, cln: &Lightningd, lnd: &Lnd) -> R
     })
     .await?;
     bitcoind.mine_blocks(10).await?;
+
+    poll("Wait for channel update", || async {
+        let mut lnd_client = lnd.client.lock().await;
+        let channels = lnd_client
+            .lightning()
+            .list_channels(ListChannelsRequest {
+                active_only: true,
+                ..Default::default()
+            })
+            .await?
+            .into_inner();
+
+        if channels.channels.is_empty() {
+            return Ok(false);
+        }
+
+        let chan_id = channels
+            .channels
+            .get(0)
+            .expect("No channels returned")
+            .chan_id;
+        let chan_info = lnd_client
+            .lightning()
+            .get_chan_info(ChanInfoRequest { chan_id })
+            .await;
+
+        if chan_info.is_err() {
+            return Ok(false);
+        }
```

Comment:
Is this expected to happen? If not we might want to log it?

Reply:
It is expected to happen, since we try to update the policy immediately after the channel is open. Rather than add an artbitrary sleep, I figured it was better to just try and then re-try if LND doesn't see the channel yet.